utils/validation.js

```diff
@@ -1,7 +1,7 @@
 const Joi = require('joi');
 
 const baseContactSchema = {
```

Comment:
Można też wykorzystać tę bibliotekę do tego aby sprawdzać czy pole jest obecne czego nie ma. Gdyby to zrobić wymagałoby to pewnych zmian prawdopodobnie w innych miejscach w kodzie ale to jest do rozważenia.